IntroOOP/AbstractFactory/src/com/globant/abstractfactory/app/SQLFactoryDemo.java

```diff
@@ -0,0 +1,41 @@
+package com.globant.abstractfactory.app;
+
+import com.globant.abstractfactory.connection.SQLConnection;
+import com.globant.abstractfactory.factory.FactoryProducer;
+import com.globant.abstractfactory.factory.SQLFactory;
+import java.util.logging.Logger;
+
```

Comment:
Comentarios

Reply:
Hello Leandro, already added the javaDocs on [this commit](https://github.com/java-bootcamp-rcia/facundo-bohle/pull/1/commits/e59083731957e0c3f9ffeb0d22d0f9a02f757b45)